assets/app/components/nav.js

```diff
@@ -12,7 +12,8 @@ const getSecondaryNav = (username) => {
   const links = [
     <Link to="/sites">{username}</Link>,
     <a href="https://federalist-docs.18f.gov" target="_blank">Documentation</a>,
-    <a href="https://github.com/18F/federalist/issues/new" target="_blank">Contact us</a>,
+    <a href="https://github.com/18F/federalist/issues/new" target="_blank">File Issue</a>,
+    <a href="mailto:federalist-support@gsa.gov" target="_blank">Get Help</a>,
```

Comment:
This looks like it should work. I don't know that you need to `target="_blank"` though?